mock-core-configs/etc/mock/templates/openeuler-20.03.tpl

```diff
@@ -28,42 +28,42 @@ user_agent={{ user_agent }}
 
 [OS]
 name=OS
-metalink=https://mirrors.openeuler.org/metalink?repo=openEuler-20.03-LTS-SP3/OS&arch=$basearch
+metalink=https://mirrors.openeuler.org/metalink?repo=$releasever/OS&arch=$basearch
```

Comment:
Mock fails for me on Fedora with this:

```
No matches found for the following disable plugin patterns: local, spacewalk, versionlock
OS                                                                                                                                                                                                          1.0 kB/s | 8.5 kB     00:08    
Errors during downloading metadata for repository 'OS':
  - Status code: 404 for https://mirrors.openeuler.org/metalink?repo=20.03-LTS/OS&arch=x86_64 (IP: 49.0.228.13)
Error: Failed to download metadata for repo 'OS': Cannot prepare internal mirrorlist: Status code: 404 for https://mirrors.openeuler.org/metalink?repo=20.03-LTS/OS&arch=x86_64 (IP: 49.0.228.13)
ERROR: Command failed: 
 # /usr/bin/dnf-3 --installroot /var/lib/mock/openeuler-20.03-x86_64-bootstrap/root/ --releasever 20.03-LTS --setopt=deltarpm=False --allowerasing --disableplugin=local --disableplugin=spacewalk --disableplugin=versionlock install python3-dnf dnf-plugins-core
No matches found for the following disable plugin patterns: local, spacewalk, versionlock
OS                                                                                                                                                                                                          1.0 kB/s | 8.5 kB     00:08    
Errors during downloading metadata for repository 'OS':
  - Status code: 404 for https://mirrors.openeuler.org/metalink?repo=20.03-LTS/OS&arch=x86_64 (IP: 49.0.228.13)
Error: Failed to download metadata for repo 'OS': Cannot prepare internal mirrorlist: Status code: 404 for https://mirrors.openeuler.org/metalink?repo=20.03-LTS/OS&arch=x86_64 (IP: 49.0.228.13)
```

I suppose you need to set `config_opts['releasever']` to an appropriate value, too.